aip/0166.md

```diff
@@ -0,0 +1,78 @@
+---
+aip:
+  id: 166
+  state: reviewing
+  created: 2020-06-24
+permalink: /166
+redirect_from:
+  - /0166
+---
+
+# Hierarchically-derived resources
+
+Services occasionally have resources where the resource's effective values are
+determined by consulting both configuration on the resource as well as
```

Comment:
I'm not sure that the two "configurations" in this sentence are useful, particularly when it's also used in the next sentence in a somewhat different way. (If it's *not* a configuration resource, what do the two "configurations" here mean?)

Reply:
Fixed.